index.html

```diff
@@ -321,6 +321,88 @@ <h2>Securing JSON-LD VCs with COSE</h2>
       </section>
     </section>
 
+
+
+<section class="normative">
+  <h2>Key Discovery</h2>
+  <p>
+In order to complete the <a data-cite="VC-DATA-MODEL#dfn-verify">verification</a> process, 
+a <a data-cite="VC-DATA-MODEL#dfn-verifier">verifier</a> needs to obtain the cryptographic keys used to secure the 
+<a data-cite="VC-DATA-MODEL#dfn-credential">credential</a>.
+  </p>
+  <p>
+There are several different ways to discover the verification keys of
+the <a data-cite="VC-DATA-MODEL#dfn-issuers">issuers</a>
+and <a data-cite="VC-DATA-MODEL#dfn-holders">holders</a>.
+  </p>
+
+  <section>
+    <h2>Registered Claim Names</h2>
+    <p>
+When found in the <a data-cite="RFC7515#section-4.1">Protected Header</a>, or 
+the <a data-cite="RFC7519#section-4.1.1">Protected Claimset</a>, members present in
+<a href="https://www.iana.org/assignments/jwt/jwt.xhtml">IANA Assignments for JSON Web Token (JWT)</a> and 
+<a href="https://www.iana.org/assignments/jose/jose.xhtml">IANA Assignments for JSON Object Signing and Encryption (JOSE)</a>
+are to be interpreted according to the associated specifications referenced by IANA.
+    </p>
+    <p>
+<a href="#registered-claim-names">Registered claims</a> that are present in either
+the <a data-cite="RFC7515#section-4.1">Protected Header</a>
+or the <a data-cite="RFC7519#section-4.1.1">Claimset</a> can be used to help
+<a data-cite="VC-DATA-MODEL#dfn-verifier">verifiers</a> discover verification keys.
+    </p>
+    <section>
+      <h2>kid</h2>
+      <p>
+If <code>kid</code> is present in the <a data-cite="RFC7515#section-4.1">Protected Header</a>,
+a <a data-cite="VC-DATA-MODEL#dfn-verifier">verifier</a> can use this parameter
+to obtain a <a data-cite="RFC7517#section-4">JSON Web Key</a> to use in the 
+<a data-cite="VC-DATA-MODEL#dfn-verify">verification</a> process.
+      </p>
+    </section>
+    <section>
+      <h2>iss</h2>
+      <p>
+If <code>iss</code> is present in the <a data-cite="RFC7515#section-4.1">Protected Header</a>
+or the <a data-cite="RFC7519#section-4.1.1">JWT Claims </a>,
+a <a data-cite="VC-DATA-MODEL#dfn-verifier">verifier</a> can use this parameter
+to obtain a <a data-cite="RFC7517#section-4">JSON Web Key</a> to use in the 
+<a data-cite="VC-DATA-MODEL#dfn-verify">verification</a> process.
+      </p>
+      <p>
+If <code>kid</code> is also present, it is expected to be useful to distinguish the specific key used.
+      </p>
+    </section>
```

Comment:
```suggestion
    <p class="issue"  data-number="31">
      There are a few issues related to clarification of optionality and behavior of 
      registered claims.  
    </P
    </section>
```

Reply:
@OR13 I am fine with merging as is so i am not marking this as change requested since we already have issues open, but referencing an exemplary issue here i think is a good idea

Reply:
looks like Ted created #117 to track his concern, if you want to use that here.

Reply:
I think I want to link all of them at this point, because people keep creating new ones.

Reply:
I will add all the links by hand when I address merge conf